modules/oe_whitelabel_search/oe_whitelabel_search.module

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * OE Whitelabel search.
+ */
+
+declare(strict_types=1);
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ *
+ * Add suggestions based on the block region where the form is placed.
+ */
+function oe_whitelabel_search_theme_suggestions_form_alter(array &$suggestions, array $variables): void {
+  if (empty($variables['element']['#form_id'])) {
```

Comment:
This is oe_whitelabel_search so lets add suggestions only to the stuff we need, if we will need it anywhere else we can move them out of this module.
Do the condition like you did for oe_whitelabel_search_theme_suggestions_input_alter, only for the templates we need